build/Common.prod.props

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
We might want to add `net7.0` to the matrix in the [linux](https://github.com/open-feature/dotnet-sdk-contrib/blob/main/.github/workflows/linux-ci.yml) and [windows](https://github.com/open-feature/dotnet-sdk-contrib/blob/main/.github/workflows/windows-ci.yml) ci workflows.